src/features/nft/nftModals/likeListModal.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import cn from 'classnames'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import Scrollbar from '../../../common/components/Scrollbar'
+// Assets
+import CaretIcon from '../../../common/assets/icons/ico-caret.svg'
+import Icon from '../../../common/components/Icon/Icon'
+
+type TDataType = {
+  id: number
+  date: string
+  userName: string
+  sold: number
+  followers: number
+}
+type TSortColumn = 'id' | 'date' | 'userName' | 'sold' | 'followers'
+type TSortOption = 'asc' | 'desc' | null
+
+type TTitle = {
+  title: string
+  name: TSortColumn
+}
+
+export type TLikeListModal = {
+  title?: string
+  content: Array<TDataType>
+  isOpen: boolean
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const titles: Array<TTitle> = [
+  { title: 'Deal', name: 'date' },
+  { title: 'Username', name: 'userName' },
+  { title: 'Sold', name: 'sold' },
+  { title: 'Followers', name: 'followers' },
+]
+
+const sortBy = (
+  data: Array<TDataType>,
+  name: TSortColumn,
+  direction: TSortOption,
+) => {
+  if (direction === 'desc') {
+    return [...data].sort((a, b) => (a[name] > b[name] ? -1 : 1))
+  } else if (direction === 'asc') {
+    return [...data].sort((a, b) => (a[name] > b[name] ? 1 : -1))
+  }
+  return data
+}
+
+const LikeListModal = ({
+  title,
+  content,
+  isOpen,
+  handleClose,
+}: TLikeListModal): JSX.Element => {
+  const [sortName, setSortName] = React.useState<TSortColumn>('date')
```

Comment:
Can we use `useState` instead of `React.useState`? Please also update for below sections